### PR TITLE
janeczku.calibre-web version 0.6.12

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25665)

### CODE REVIEW

File: manifests/j/janeczku/calibre-web/0.6.12/janeczku.calibre-web.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: janeczku.calibre-web
+PackageVersion: 0.6.12
```
Comment (ItzLevvie) [inline manifests/j/janeczku/calibre-web/0.6.12/janeczku.calibre-web.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.6.12 Beta
```

File: manifests/j/janeczku/calibre-web/0.6.12/janeczku.calibre-web.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: janeczku.calibre-web
+PackageVersion: 0.6.12
```
Comment (ItzLevvie) [inline manifests/j/janeczku/calibre-web/0.6.12/janeczku.calibre-web.yaml:5]:
```suggestion
PackageVersion: 0.6.12 Beta
```

File: manifests/j/janeczku/calibre-web/0.6.12/janeczku.calibre-web.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: janeczku.calibre-web
+PackageVersion: 0.6.12
```
Comment (ItzLevvie) [inline manifests/j/janeczku/calibre-web/0.6.12/janeczku.calibre-web.installer.yaml:5]:
```suggestion
PackageVersion: 0.6.12 Beta
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by arjun-msft** [comment]
@oxygen-dioxide Thanks for your submission! After installation, when I try to run calibreweb.exe a command window opens and then closes right away, running from the command line directly I see the following error, can you have a look?

image

**Pull request closed without merging**